docs/examples.md

```diff
@@ -1,48 +1,47 @@
 # Examples
 
-This Example has been taken with a Ledger nano S.
+Example using a Ledger Nano S:
 
 ```
 ./hwi.py enumerate
 [{"type": "ledger", "path": "IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS02@14200000/Nano S@14200000/Nano S@0/IOUSBHostHIDDevice@14200000,0", "serial_number": "0001"}, {"type": "ledger", "path": "IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/XHC1@14/XHC1@14000000/HS02@14200000/Nano S@14200000/Nano S@1/IOUSBHostHIDDevice@14200000,1", "serial_number": "0001"}]
 ```
-The OS in this case is OSX (17.7.0 Darwin Kernel Version 17.7.0). In Linux the 
+The OS in this case is macOS v10.13.6  (Darwin Kernel Version 17.7.0). In Linux the
 "path" is shorter.
 
-## Useful xpubs to extract
+## Extracting xpubs
 
-Starting from version Bitcoin Core v.0.17, It is possible to to retrieve 
-the Unspent transaction outputs relevant for a set of [Output Descriptors][1] 
-From Bitcoin Core with the `scantxoutset` funcion.
+Bitcoin Core v0.17[¹](#bitcoin-core-version) allows you to retrieve the unspent transaction outputs (utxo)
```

Comment:
Please don't use superscripts.

Reply:
I'll address this after we know if footnotes are being removed or not.

Reply:
I went ahead and replaced the superscript numbers with normal numbers.